src/app/views/media-coverage/media-coverage.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="row justify-content-center d-flex align-items-center bg-white w-100" style="max-width: 100vw;">
+
+  <div class="d-flex mt-5" style="margin-left: 25vw;width: 100vw;">
+    <h1 class="float-start">Media Coverage</h1>
+  </div>
+  <hr>
+  <div id="media-coverage" class="p-5 d-flex row justify-content-center col-md-9 pt-5" style="max-height: 70vh;">
+  <ng-container *ngFor="let item of media_coverage;index as index">
+      <div class="row shadow border card border-1 mb-2 col-12 border-bottom-3 border-dark d-flex justify-content-center p-5" >
+          <h4 class="col"><b>{{item.Title}}</b></h4>
+          <h5 class="col" >{{item.Details}}</h5><br>
+          <div id={{index}} class="row py-4 col-md-7 col-12 collapse">
+        </div>
+        <a href= {{item.link[0]}}>click here</a>
```

Comment:
add "target='_blank' " in this line.
Capitalize text Click Here or change with a better one like => Click to view fulll story